File: api/code-samples/python/eth/filter/getFilterLogs.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

quantity = "0x11b5f0ef802c98569efa8459cf063ed5"

sdk = OpenSDK(host)
eth_response = sdk.eth.get_filter_logs(quantity)

print(json.loads(eth_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/klay/filter/getFilterLogs.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const quantity = "0x16"
    sdk.klay.getFilterLogs(quantity, {}, (err, data, response) => {
        console.log(data);
    });
}
)()